src/app/components/feature-toggling/feature-toggling.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { BreadcrumbLink } from '../generics/navigation-breadcrumb/navigation-breadcrumb.component';
+import {
+  QcAtlasUiConfiguration,
+  QcAtlasUiRepositoryConfigurationService,
+  UiFeatures,
+} from '../../directives/qc-atlas-ui-repository-configuration.service';
+import { environment } from '../../../environments/environment';
+import { UtilService } from '../../util/util.service';
+
+@Component({
+  selector: 'app-feature-toggling',
+  templateUrl: './feature-toggling.component.html',
+  styleUrls: ['./feature-toggling.component.scss'],
+})
+export class FeatureTogglingComponent implements OnInit {
+  links: BreadcrumbLink[] = [{ heading: '', subHeading: '' }];
+
+  readonly UiFeatures = UiFeatures;
+  config: QcAtlasUiConfiguration;
+
+  constructor(
+    private http: HttpClient,
+    private configData: QcAtlasUiRepositoryConfigurationService,
+    private utilService: UtilService
+  ) {}
+
+  ngOnInit(): void {
+    this.config = this.configData.configuration;
+
+    this.links[0] = {
+      heading: 'Feature Selection',
+      subHeading: 'Set the visibility of supported features',
+    };
+  }
+
+  toggleFeature(feature): void {
+    let url = environment.CONFIG_SEVER_URL + '/features/';
```

Comment:
I guess it would be easier to do something like `const url = environment.CONFIG_SEVER_URL + '/features/' + feature;`.
Then you don't need to do the switch...